gtsam_unstable/linear/ActiveSetSolver-inl.h

```diff
@@ -165,14 +165,13 @@ Template JacobianFactor::shared_ptr This::createDualFactor(
  *  if lambda = 0  you are on the constraint
  *  if lambda > 0  you are violating the constraint.
  */
-Template GaussianFactorGraph::shared_ptr This::buildDualGraph(
+Template GaussianFactorGraph This::buildDualGraph(
```

Comment:
Is this "copy elision" / "return value optimization" since C++11?  I previously expected this would be a performance hit and just want to make sure.

Reply:
Indeed, RVO. Older than c++11 I think.